Code/GraphMol/SubstructLibrary/SubstructLibrary.h

```diff
@@ -28,20 +28,22 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
+#ifndef RDK_SUBSTRUCT_LIBRARY
+#define RDK_SUBSTRUCT_LIBRARY
 #include <RDGeneral/export.h>
-#ifndef RDKIT_SUBSTRUCT_LIBRARY
-#define RDKIT_SUBSTRUCT_LIBRARY
-
 #include <GraphMol/RDKitBase.h>
 #include <GraphMol/MolPickler.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
 #include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/Fingerprints/Fingerprints.h>
 #include <DataStructs/ExplicitBitVect.h>
 #include <DataStructs/BitOps.h>
+#include <RDGeneral/RDConfig.h>
```

Comment:
`RDConfig.h` is primarily intended for use in "client" (i.e. non-RDKit core) code.
Using it results in a ton of warnings.

Reply:
Hm, if we could remove the warnings, would this be okay?

Reply:
I don't think there's anything inherently wrong with it, but it is completely inconsistent with the rest of the code. Since this is one of those tricky things, I'd really prefer that we remain consistent.